Tools/CISupport/build-webkit-org/config.json

```diff
@@ -163,7 +163,7 @@
                   "triggers": [
                       "sequoia-release-tests-wk1", "sequoia-release-tests-wk2", "sequoia-release-perf-tests",
                       "sequoia-release-applesilicon-tests-wk1", "sequoia-release-applesilicon-tests-wk2",
-                      "sequoia-applesilicon-release-tests-test262", "sequoia-release-tests-wk2-accessibility-isolated-tree"
+                      "sequoia-applesilicon-release-tests-test262", "sequoia-release-tests-wk2-accessibility-isolated-tree", "sequoia-release-tests-wk2-site-isolation-tree"
```

Comment:
style nit: I don't think we're truly enforcing line length in this file, but it might be cleaner to put this on a new line by itself for readability.

Reply:
Done!